cal/cal/static/js/app.js

```diff
@@ -15,37 +15,25 @@ function TagListCtrl($scope, $http, CalendarFilterService, TagService) {
 
   $scope.$on('calendarFilter:updated', function(event, data) {
     /* jshint unused:vars */
+    _this.tags.dataLoaded = false;
     var filterData = CalendarFilterService.getFilter();
-    if (!this.isCumulative) {
+    if (!_this.isCumulative) {
       TagService.getTags(filterData.filterKey, filterData.start, filterData.end,
                          filterData.calendarIds)
         .then(function(tags) {
-          this.tags = tags;
-          this.filterKey = filterData.filterKey;
-          this.tags.dataLoaded = true;
+          _this.tags = tags;
+          _this.filterKey = filterData.filterKey;
+          _this.tags.dataLoaded = true;
         });
-      }
-  }.bind(this));
-
-  // Initialization
-  this.initialize = function() {
-    var tagsPromise;
-    var initialFilterData = CalendarFilterService.getFilter();
-    if (this.isCumulative) {
-      tagsPromise = TagService.getTags('cumulative', null, null,
-        initialFilterData.calendarIds);
     } else {
-      tagsPromise = TagService.getTags(initialFilterData.filterKey,
-        initialFilterData.start, initialFilterData.end,
-        initialFilterData.calendarIds);
+      TagService.getTags('cumulative' + filterData.filterKey, null, null,
```

Comment:
Would you mind adding a space after the `e` so that it's `'cumulative '`? The other parts of the filterKey are currently separated by spaces.